Add West Old Turkic concept list

martino-vic opened the pull request:
# Pull request checklist

- [x] add new concept list
- [x] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [x] add new Concepticon concept relations
- [x] refine existing Concepticon concept set mappings
- [x] refine Concepticon glosses
- [x] refine Concepticon concept relations
- [x] refine Concepticon concept definitions
- [ ] retire data

## Additional information
I've followed the instructions mentioned in [this issue](https://github.com/concepticon/concepticon-data/issues/1180)
I didn't add new Concepticon concept sets, as that was the outcome of [this issue](https://github.com/concepticon/concepticon-data/issues/1238)
I'm not sure what the last four bullets mean, would be glad if somebody could give me a pointer
Also, when I run `conception test` I get `ValueError: invalid Conceptset.semanticfield: ['Agriculture and Vegetation']`, I couldn't decipher the rest of the error message either, which is:

INFO    concepticon/concepticon-data at /home/viktor/Documents/GitHub/concepticon-data
testing 410 concept lists
Traceback (most recent call last):
  File "/home/viktor/Documents/cldfvenv3.9/bin/concepticon", line 8, in <module>
    sys.exit(main())
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/pyconcepticon/__main__.py", line 62, in main
    return args.main(args) or 0
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/pyconcepticon/commands/test.py", line 21, in run
    if args.repos.check(*args.clids):  # pragma: no cover
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/pyconcepticon/api.py", line 391, in check
    'concepticon_id': set(self.conceptsets.keys()),
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/clldutils/misc.py", line 197, in __get__
    result = instance.__dict__[self.__name__] = self.fget(instance)
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/pyconcepticon/api.py", line 120, in conceptsets
    return to_dict(
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/pyconcepticon/util.py", line 36, in to_dict
    for obj in iterobjects:
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/pyconcepticon/api.py", line 121, in <genexpr>
    Conceptset(api=self, **lowercase(d))
  File "<attrs generated init pyconcepticon.models.Conceptset>", line 10, in __init__
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/pyconcepticon/models.py", line 69, in valid_key
    raise ValueError('invalid {0}.{1}: {2}'.format(
ValueError: invalid Conceptset.semanticfield: ['Agriculture and Vegetation']
...

AnnikaTjuka commented:
The devil is in the details: The semantic field tag needs to be changed from "Agriculture and Vegetation" to "Agriculture and vegetation".

I only see the file `concepticon.tsv` in the tab "Files changed". Did you want to discuss the error or add a new list? If the latter, you can check out an old PR from @mathildavz to see which files need to be uploaded before we can start the review process. https://github.com/concepticon/concepticon-data/pull/1188

martino-vic commented:
Okay strange, mein GitHub desktop hatte die Änderungen nicht gepusht, jetzt sind die 5 Files geändert. Sorry für den blooper.

Bessere gleich das "V" zu "v" aus

xrotwang commented:
From the few interactions I had with GitHub desktop, I found it not very helpful, in particular when it comes to diagnosing bugs/unexpected situations. Using git on the command line might make this more transparent.

AnnikaTjuka commented:
Three more issues:
- The description of the list in `conceptlists.tsv` is incomplete.
- Some columns have quotation marks for the entries. This is usually caused by Excel or LibreOffice. I recommend opening lists only with a text editor (or Google Sheets when working together on a list). I use [Sublime Text](https://www.sublimetext.com/), but you can also use another text editor.
- The result of the discussion in issue #1238 was not to add any new concept sets. Thus, the file `concepticon.tsv` should not be updated in this PR.

One question:
- Why do so many glosses occur more than once?

martino-vic commented:
So now I have:
- completed the description
- removed quotation marks
- removed the 20 new concept sets
- removed glosses that occurred more than once (they were an accident)

Getting following error message when running `concepticon test`:

INFO    concepticon/concepticon-data at /home/viktor/Documents/GitHub/concepticon-data
testing 410 concept lists
Traceback (most recent call last):
  File "/home/viktor/Documents/cldfvenv3.9/bin/concepticon", line 8, in <module>
    sys.exit(main())
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/pyconcepticon/__main__.py", line 62, in main
    return args.main(args) or 0
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/pyconcepticon/commands/test.py", line 21, in run
    if args.repos.check(*args.clids):  # pragma: no cover
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/pyconcepticon/api.py", line 423, in check
    for col in cl.metadata.tableSchema.columns:
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/clldutils/misc.py", line 197, in __get__
    result = instance.__dict__[self.__name__] = self.fget(instance)
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/pyconcepticon/models.py", line 234, in metadata
    return self.tg.tables[0]
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/clldutils/misc.py", line 197, in __get__
    result = instance.__dict__[self.__name__] = self.fget(instance)
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/pyconcepticon/models.py", line 226, in tg
    tg = TableGroup.from_file(md)
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/csvw/metadata.py", line 565, in from_file
    res = cls.fromvalue(data)
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/csvw/metadata.py", line 202, in fromvalue
    return cls(**cls.partition_properties(d))
  File "<attrs generated init csvw.metadata.TableGroup>", line 39, in __init__
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/csvw/metadata.py", line 808, in <lambda>
    converter=lambda v: [Table.fromvalue(vv) for vv in v])
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/csvw/metadata.py", line 808, in <listcomp>
    converter=lambda v: [Table.fromvalue(vv) for vv in v])
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/csvw/metadata.py", line 202, in fromvalue
    return cls(**cls.partition_properties(d))
  File "<attrs generated init csvw.metadata.Table>", line 31, in __init__
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/csvw/metadata.py", line 545, in <lambda>
    converter=lambda v: Schema.fromvalue(v))
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/csvw/metadata.py", line 517, in fromvalue
    return cls(**cls.partition_properties(v))
  File "<attrs generated init csvw.metadata.Schema>", line 26, in __init__
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/csvw/metadata.py", line 491, in <lambda>
    converter=lambda v: [Column.fromvalue(c) for c in v])
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/csvw/metadata.py", line 491, in <listcomp>
    converter=lambda v: [Column.fromvalue(c) for c in v])
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/csvw/metadata.py", line 202, in fromvalue
    return cls(**cls.partition_properties(d))
  File "<attrs generated init csvw.metadata.Column>", line 12, in __init__
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/csvw/metadata.py", line 362, in <lambda>
    converter=lambda v: v if not v else Datatype.fromvalue(v))
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/csvw/metadata.py", line 268, in fromvalue
    return cls(**DescriptionBase.partition_properties(v))
  File "<attrs generated init csvw.metadata.Datatype>", line 22, in __init__
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/attr/validators.py", line 269, in __call__
    self.validator(inst, attr, value)
  File "/home/viktor/Documents/cldfvenv3.9/lib/python3.9/site-packages/attr/validators.py", line 306, in __call__
    raise ValueError(
ValueError: 'base' must be in {'any': <class 'csvw.datatypes.anyAtomicType'>, 'string': <class 'csvw.datatypes.string'>, 'anyURI': <class 'csvw.datatypes.anyURI'>, 'binary': <class 'csvw.datatypes.base64Binary'>, 'hexBinary': <class 'csvw.datatypes.hexBinary'>, 'boolean': <class 'csvw.datatypes.boolean'>, 'datetime': <class 'csvw.datatypes.dateTime'>, 'date': <class 'csvw.datatypes.date'>, 'dateTimeStamp': <class 'csvw.datatypes.dateTimeStamp'>, 'time': <class 'csvw.datatypes._time'>, 'duration': <class 'csvw.datatypes.duration'>, 'decimal': <class 'csvw.datatypes.decimal'>, 'integer': <class 'csvw.datatypes.integer'>, 'nonNegativeInteger': <class 'csvw.datatypes.nonNegativeInteger'>, 'float': <class 'csvw.datatypes._float'>, 'number': <class 'csvw.datatypes.number'>, 'double': <class 'csvw.datatypes.double'>, 'QName': <class 'csvw.datatypes.QName'>, 'gDay': <class 'csvw.datatypes.gDay'>, 'gMonth': <class 'csvw.datatypes.gMonth'>, 'gMonthDay': <class 'csvw.datatypes.gMonthDay'>, 'gYear': <class 'csvw.datatypes.gYear'>, 'gYearMonth': <class 'csvw.datatypes.gYearMonth'>, 'xml': <class 'csvw.datatypes.xml'>, 'html': <class 'csvw.datatypes.html'>, 'json': <class 'csvw.datatypes.json'>} (got 'positiveInteger')

xrotwang commented:
`positiveInteger` was only added to `csvw` recently (see https://github.com/cldf/csvw/blame/master/src/csvw/datatypes.py#L763-L769). So it looks like you'd need to update your `csvw` package.

xrotwang commented:
Looks like the software does what it's supposed to do: pointing out problems with the data. Do you think the reported problem is a false positive?

martino-vic commented:
I'm having troubles understanding what the error message is trying to tell me.
My first question:
Which file contains the error? does `ERROR:RonaTas-2011-431:` mean the error is in `RonaTas-2011-431.tsv`?

What I so far understand is that some file somewhere in the repository is not a string or byte-like object. But which one that could be? There are three tsv-files, one json, and one .bib

xrotwang commented:
That's a bit tricky to debug, I admit. But `NUMBER` is acutally checked against a regular expression, here:
https://github.com/concepticon/pyconcepticon/blob/5ce8ad7944f408c006c1014a5c6ca532305dd6bb/src/pyconcepticon/models.py#L119-L120
Since you defined `NUMBER` as `integer`, this check raises a `TypeError`, the stack trace of which is unfortunately swallowed here
https://github.com/concepticon/pyconcepticon/blob/5ce8ad7944f408c006c1014a5c6ca532305dd6bb/src/pyconcepticon/api.py#L486-L488

martino-vic commented:
Cool, thanks for the help! Now the tests are passing:

INFO    concepticon/concepticon-data at /home/viktor/Documents/GitHub/concepticon-data
testing 410 concept lists
INFO    all integrity tests passed: OK

Pointed me to two wrong entries, that worked well.

AnnikaTjuka commented:
Ok, great! It looks like the PR is ready for our usual review process of checking the mappings and metadata. @mathildavz and @Tarotis Could you be a reviewer for this PR?

Tarotis commented on concepticondata/references/references.bib:
```diff
@@ -4266,5 +4266,12 @@ @Article{Nordenskioeld1905
+  Year                                = {1905}
+}
+
+@Article{RonaTas2011,
+  Author                              = {Róna-Tas, András, and Berta, Árpád},
+  Title                               = {West Old Turkic},
+  Publisher                           = {Harrassowitz},
+  Year                                = {2011}
```
The bibliography misses information on address (Wiesbaden) and the subtitle (Turkic loanwords in Hungarian). Could you add those?

Tarotis replied:
Oh yes, the most important change: The bibtype is `Book`, not `Article`. Thanks for pointing that out.

martino-vic replied:
Implemented the changes with d8aeaad31eb06b7c9165d9202b5c5c5fa1bc2744

AnnikaTjuka replied:
> Implemented the changes with [d8aeaad](https://github.com/concepticon/concepticon-data/commit/d8aeaad31eb06b7c9165d9202b5c5c5fa1bc2744)

We see that the changes are implemented with the note "outdated", so you do not need to add the comment above. You can use the "Resolve conversation" button so that the comments are collapsed. I'll make a final check at the end of the review to see if all changes are correctly implemented.

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+RonaTas-2011-431-1	1	carpenter	1042	CARPENTER
+RonaTas-2011-431-2	2	cannon, catapult	3202	CANNON
+RonaTas-2011-431-3	3	to fondle, pet, caress	3780	PET
```
`PET` is usually mapped to the noun, not to the verb. I would unmap

martino-vic replied:
Resolved in 147dea165b7393d78c894139004de111860c83a2

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-6	6	pied (of an animals coat)		
+RonaTas-2011-431-7	7	apple	1320	APPLE
+RonaTas-2011-431-8	8	barrel (wooden container andstandard quantity)	2548	BARREL
+RonaTas-2011-431-9	9	to nurse, take care of	2472	SUCKLE (BREASTFEED, NURSE)
```
I would possibly also unmap this. The meaning in the original list seems to refer more towards an educational nursing (something like german "großziehen"), while `SUCKLE` is quite specific for breastfeeding.

martino-vic replied:
Resolved in 029e10648eb0e8351eec8b62d2130cb9250afc10

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-7	7	apple	1320	APPLE
+RonaTas-2011-431-8	8	barrel (wooden container andstandard quantity)	2548	BARREL
+RonaTas-2011-431-9	9	to nurse, take care of	2472	SUCKLE (BREASTFEED, NURSE)
+RonaTas-2011-431-10	10	to decay, putrify, turn stale	2907	ROT
```
If we enforce unique mappings, I'd probably unmap this `ROT` instead of the second one (ID 13).

martino-vic replied:
Resolved in d5237b728dd839b9352f77f30508546e248f53f4

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-15	15	a type of lasso		
+RonaTas-2011-431-16	16	ditch, canal	2002	DITCH
+RonaTas-2011-431-17	17	barley, Hordeum	932	BARLEY
+RonaTas-2011-431-18	18	barrow	2565	CASTRATED BOAR
```
I didn't knew the word "barrow", but it seems to be the german "Hügelgrab" or  "Schubkarre". So I'd unmap.

AnnikaTjuka replied:
I agree. I found this description for looking up "Old English barrow": Barrows are useful for carrying loads of things, like dirt or leaves or sand, from one place to another.

Tarotis replied:
Oh, there is a Hungarian original? Then it could even make sense to add this as an extra column. Or do the authors provide the English concept as well?

martino-vic replied:
Yeah, this is how the entry looks like:

![Screenshot from 2022-10-11 12-01-36](https://user-images.githubusercontent.com/53435298/195063211-ceddeee3-3ed0-48fe-845a-359a80b8f595.png)

![Screenshot from 2022-10-11 12-01-50](https://user-images.githubusercontent.com/53435298/195063192-fa224514-f48d-4b7d-b58d-0873e14097e7.png)

So they provide the translation already, but when in doubt you can always look up the Hungarian word in some other dictionary as well (like I did with dict.cc and wiktionary in the previous comment)

martino-vic replied:
Idk, should I add an extra column with the Hungarian words? Haven't thought about that yet.

Tarotis replied:
I am not sure, as they also provide the translation themselves. On the other hand, it clearly helps disambiguating the cases. @AnnikaTjuka what do you think?

AnnikaTjuka replied:
I assumed it is a concept list, not an etymological items list (similar to [Zhang-2019-308](https://concepticon.clld.org/contributions/Zhang-2019-308)). I see that adding the list has already been discussed in several issues, so I would like to ask @LinguList to make the final decision here.

LinguList replied:
Let us leave out the Hungarian for now. It will be available in the lexibank data, and in this way, we really reduce it to a concept list -- or hope to do so.

martino-vic replied:
Alright, then I will close this convo now.

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-20	20	charm		
+RonaTas-2011-431-21	21	soggy place, moor, swamp, marsh	1145	SWAMP
+RonaTas-2011-431-22	22	hatchet, axe	677	AXE
+RonaTas-2011-431-23	23	pagan idol, stone or wooden pillar, vertical beam of different wooden instruments, strong, corpulent person	857	STONE
```
`STONE` refers to english "stone", while this entry seems to be `1945 IDOL`. I'd change the mapping.

martino-vic replied:
resolved in 3288105d13e2264f434aa08790bfaaebcd5e3458

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-27	27	label, tag		
+RonaTas-2011-431-28	28	cattle, livestock	1153	CATTLE
+RonaTas-2011-431-29	29	velvet		
+RonaTas-2011-431-30	30	to fuck		
```
`549 COPULATE`, this can be mapped

martino-vic replied:
Ah, I deliberately left this unmapped because I thought from a pragmatic point of view those concepts are quite different. Also I feel like the meaning is somehow narrower? Ah, just checked with [WordNet](http://wordnetweb.princeton.edu/perl/webwn?o2=&o0=1&o8=1&o1=1&o7=&o5=&o9=&o6=&o3=&o4=&s=fuck&i=4&h=01000000#c) and they also list "to fuck" as a **hypernym** of "to copulate"

LinguList replied:
The word never occurred in this form in our data. It depends on your judgment of what you think the authors wanted to denote: a rude, tabooised version of "to copulate" or did they just use it in the way that people also write "shit", to make sure one does not confuse it when they use alternative glosses or that they want to avoid formal-sounding language? If you decide to not map it, you could add the `ignore.tsv` file for the first time, place it in `concepticondata`, add the ID of the entry, the gloss, and a comment, why it is not "to copulate".

martino-vic replied:
Alright, will try to add the `ignore.tsv`, sounds like a good exercise.

martino-vic replied:
Resolved in 9a25ac393582ec7d459ea2e5ad798b277c1ec186

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-33	33	frog	503	FROG
+RonaTas-2011-431-34	34	hobble, shackle, leg-iron		
+RonaTas-2011-431-35	35	stamp, bond, mark	1720	SIGN
+RonaTas-2011-431-36	36	wage, rent		
```
This could be mapped to `1122 WAGES`.

AnnikaTjuka replied:
I'd leave it unmapped since WAGES is mapped to glosses like _Lohn_, _salary_, and not _rent_.

martino-vic replied:
Alright, will leave it unmapped then, just in case

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-75	75	Ismaelite, name of an ethnic group in Medieval Hungary		
+RonaTas-2011-431-76	76	sad, bad-tempered	699	SAD
+RonaTas-2011-431-77	77	farewell, indulgence, dedication festival, annual parish-feast	3554	GOODBYE
+RonaTas-2011-431-78	78	ox; also ox, goat, ram without or with small or unusual horns;stupid	1502	GOAT
```
I'd prefer to map to `1169 OX`, as this seems to be the principle meaning

AnnikaTjuka replied:
This seems to denote a concept like ANIMAL WITH HORNS so it is too broad to be mapped to OX.

martino-vic replied:
From the original source: "The H[ungarian] word is well represented in the H[ungarian] dial[ect]s of Central and Western Hungary, but the first attestation as a common noun is fixed relatively late, in 1775, on a wordlist of the slang of a certain thief (Schram 1962: 108). As a PN it first occurs in 1215 (copy 1550) and later (Fehértói 2004: 157)"

I'll leave it unmapped, since it's a dialectal word. Would need to check Fehértói 2004: 157 to read more about the meaning

martino-vic replied:
Resolved in 98a7e57a41cbe42bd46bc878ec2850518c4988b5

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-89	89	a kind of willow, used for baskets and in viticulture, Salix purpurea		
+RonaTas-2011-431-90	90	a kind of willow		
+RonaTas-2011-431-91	91	a kind of wolfs milk, Easter flower, Euphorbia		
+RonaTas-2011-431-92	92	a fish, Leuciscus souffia agassizi	227	FISH
```
As this is a specific fish, I would unmap

martino-vic replied:
Resolved in f7c1e0b122787d0d6722869d5b6b4a6dc23d6600

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-94	94	pick-axe; mace; a small axe; war hammer		
+RonaTas-2011-431-95	95	nettle, Urtica	33	NETTLE
+RonaTas-2011-431-96	96	thicket, brushwood, dry foliage, grove, meadow	2819	GRASSLAND
+RonaTas-2011-431-97	97	thicket, brushwood; dry foliage, grove, meadow	2819	GRASSLAND
```
This seems like a duplicate entry and should be removed

AnnikaTjuka replied:
If it is in the original list, I would leave it as is. @martino-vic Could you double-check with the original list?

martino-vic replied:
Just checked in the original source and it's treated as two separate cognate sets because of a possible metathesis. So one is `t͡ʃ ɒ l i t <  t͡ʃ a l t ɯ`, the other `t͡ʃ ɒ l i t < t͡ʃ a l ɯ t`. See https://github.com/martino-vic/rtbwestoldturkic/blob/main/cldf/forms.csv#L376

martino-vic replied:
Partially fixed this in 0ca3d47d736423dc3eb770614d9a9b3f5a89c47e
Since libre office, excel and google sheets all do unexpected formating (quotations marks, turning numbers into dates, autranslating words like "FALSE" to "FALSCH"), I have to use some other tool to re-numerate this column. Will try with pandas once I'm through editing all issues

AnnikaTjuka replied:
If the original list contains this line, I would leave it in. We want to reflect the original list as accurately as possible, and someone may want to look up the two cognate sets at some point.

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-96	96	thicket, brushwood, dry foliage, grove, meadow	2819	GRASSLAND
+RonaTas-2011-431-97	97	thicket, brushwood; dry foliage, grove, meadow	2819	GRASSLAND
+RonaTas-2011-431-98	98	(silver) cup, wooden ladle, earthenware bowl	476	CUP
+RonaTas-2011-431-99	99	fighting, troop, army group, battle, quarrel, hubbub, flock	1848	QUARREL
```
The meaning of this seems too broad in order to map it to `QUARREL`. Could equally be mapped to `ARMY`, so we should probably leave it without any mapping

martino-vic replied:
Resolved in 70712118b994f4bb5bc92d2a83a1e620424092b6

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-116	116	rough, coarse; ugly; wild, unbridled	1923	ROUGH
+RonaTas-2011-431-117	117	to wind, to turn, to distrort, misinterpret, to spin a yarn	1588	TURN
+RonaTas-2011-431-118	118	bobbin, spindle	277	SPINDLE
+RonaTas-2011-431-119	119	excellent; worthless		
```
Here I am mainly surprised by the different meanings that are added in one entry. Can you confirm that this is the original entry?

martino-vic replied:
"The word daksi is known nowadays, as pointed out by Mándoky, in almost all the settlements of Cumania Major in Hungary not only in the sense of ‘perfect, outstanding, very good’, but also with the opposite meaning ‘bad, worthless’, e.g. in H daksi bunda [dåkši bundå] ‘ragged fur-coat’." (p.283)

Afaik this is a quite common phenomenon, just think of English <terrific>

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-119	119	excellent; worthless		
+RonaTas-2011-431-120	120	a short fur coat		
+RonaTas-2011-431-121	121	grist, groats, soft hail		
+RonaTas-2011-431-122	122	noon, south	12	MIDDAY
```
I would unmap, could equally be mapped to `SOUTH`. Or do we take the first meaning as a kind of "principal meaning" @AnnikaTjuka ?

AnnikaTjuka replied:
We take both glosses into account unless the author explicitly states that there is a "main meaning". So I would also unmap here.

martino-vic replied:
Resolved in cef31d4c9006252a039261afdabfc3b3f1303ff7

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-122	122	noon, south	12	MIDDAY
+RonaTas-2011-431-123	123	nut; walnut, Juglans	2042	WALNUT
+RonaTas-2011-431-124	124	pig, Sus domestica	1337	PIG
+RonaTas-2011-431-125	125	to lean, topple over, stream down	3619	LEAN
```
I'd unmap, as this includes many different meanings.

martino-vic replied:
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-133	133	to arrive, to reach, get to	1387	ARRIVE
+RonaTas-2011-431-134	134	merit		
+RonaTas-2011-431-135	135	morals, morality		
+RonaTas-2011-431-136	136	to tire, lose vitality, slacken, relax	168	REST
```
The main meaning seems to be `1996 GET TIRED`, I'd change the mapping or unmap due to the presence of "relax"

AnnikaTjuka replied:
I'd also unmap.

martino-vic replied:
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-143	143	matchmaker		
+RonaTas-2011-431-144	144	gleaner mouse, Mus spicilegus		
+RonaTas-2011-431-145	145	to abuse, calumniate	1953	ABUSE
+RonaTas-2011-431-146	146	drag-net, a kind of fishing net		
```
Are there many different fish nets? I think this could possibly be mapped to `388 FISHNET`

AnnikaTjuka replied:
I would assume that NETBAG (a net for carrying) and FISHNET are meant. I'd unmap.

martino-vic replied:
the original Hungarian word is \<gyalom\> "Zeese" (https://dictzone.com/ungarisch-deutsch-worterbuch/gyalom and https://en.wikipedia.org/wiki/Zeese), so I'll unmap

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-166	166	cudweed, Gnaphalium; Edelweiss		
+RonaTas-2011-431-167	167	pearl	3325	PEARL
+RonaTas-2011-431-168	168	to torture, to make suffer	1952	TORTURE
+RonaTas-2011-431-169	169	to conquer (somebody, something), triumph (over somebody, something), gain victory	3849	TRIUMPH
```
`TRIUMPH` maps to the noun, not the verb, so I'd unmap

martino-vic replied:
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-169	169	to conquer (somebody, something), triumph (over somebody, something), gain victory	3849	TRIUMPH
+RonaTas-2011-431-170	170	to catch fire, be kindled, to be ignited		
+RonaTas-2011-431-171	171	to knead, pug	274	KNEAD
+RonaTas-2011-431-172	172	to assemble, gather	7	GATHER
```
I'd change the mapping to `3638 ASSEMBLE (GET TOGETHER)`

AnnikaTjuka replied:
A similar gloss is mapped to GATHER in [Huang-1992-1820-1402](https://concepticon.clld.org/values/Huang-1992-1820-1402). @martino-vic Can you disambiguate the gloss? Otherwise, I'd unmap.

martino-vic replied:
The original word \<gyűlik\> mostly means "to gather". I think the other meaning is again there to explain the semantic processes underlying the etymology

martino-vic replied:
Resolved in 2c64368e0fcb7cc87964075b2696cbf49d3f68ba

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-290	290	ox	1169	OX
+RonaTas-2011-431-291	291	to stitch, to put on a dress		
+RonaTas-2011-431-292	292	hawk, buzzard, Buteo buteo	307	HAWK
+RonaTas-2011-431-293	293	to take root, to grow stronger (plant, animal)	619	ANIMAL
```
I'd unmap, this does not refer to `ANIMAL`

martino-vic replied:
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-349	349	part of a village, street, group of people; row, line, order	1362	STREET
+RonaTas-2011-431-350	350	threshing floor (round)		
+RonaTas-2011-431-351	351	fumes, vapour, spirits, alcohol, humours	404	ALCOHOL (FERMENTED DRINK)
+RonaTas-2011-431-352	352	colour, face, external appearance, the best part of something, the upperpart of something	1560	FACE
```
I'd unmap due to the variety of meanings

martino-vic replied:
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-351	351	fumes, vapour, spirits, alcohol, humours	404	ALCOHOL (FERMENTED DRINK)
+RonaTas-2011-431-352	352	colour, face, external appearance, the best part of something, the upperpart of something	1560	FACE
+RonaTas-2011-431-353	353	thin hide rope, strap (used for embroidery or as a whip), refrozen thin layer after the melting of snow	2973	STRAP
+RonaTas-2011-431-354	354	rock, cliff, occiput, crest (of mountain)	992	CLIFF
```
I'd unmap due to the variety of meanings

martino-vic replied:
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-183	183	sand	671	SAND
+RonaTas-2011-431-184	184	noose, loop, snare	2604	NOOSE
+RonaTas-2011-431-185	185	time, weather	892	TIME
+RonaTas-2011-431-186	186	word, biblical word, verb	1599	WORD
```
I'd unmap due to the variety of meanings, the concept `WORD` is less ambigously defined later. This seems to be the "biblical word"

martino-vic replied:
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-386	386	hall, chamber, great room		
+RonaTas-2011-431-387	387	camel	1871	CAMEL
+RonaTas-2011-431-388	388	hemp breaker, swingle, scutch		
+RonaTas-2011-431-389	389	steer, young bullock, ox	1169	OX
```
I'd map due to the variety of meanings. Also, `OX` is already mapped less ambigously

martino-vic replied:
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-85	85	(blunt) end of something, a rod with an end cut, obtuse end of a field-strip, the shorter side or end of a piece of furniture, foot of the hill		
+RonaTas-2011-431-86	86	magical		
+RonaTas-2011-431-87	87	a kind of willow, Salix elægnos		
+RonaTas-2011-431-88	88	trap; snare (for catching birds); bird cage	1075	TRAP (PITFALL)
```
This looks like a bird trap, which is not `TRAP (PITFALL`. A more specific entry for this comes later, so I'd unmap this one

martino-vic replied:
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-403	403	to happen, to occur	248	HAPPEN
+RonaTas-2011-431-404	404	law	1062	LAW
+RonaTas-2011-431-405	405	peat, turf, dried dung		
+RonaTas-2011-431-406	406	steer, young bullock, ox or cow	1007	COW
```
Is this a duplicate to 389? In any case, I'd unmap for the same reasons as above.

martino-vic replied:
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-422	422	hot embers, extinguished embers		
+RonaTas-2011-431-423	423	one-year-old female sheep		
+RonaTas-2011-431-424	424	to hollow out, to scoop out	46	HOLLOW OUT
+RonaTas-2011-431-425	425	trough, tray	205	TROUGH
```
This entry refers to `2629 TRAY`, so I'd change the mapping

AnnikaTjuka replied:
We have two concepts for each of the glosses, so I'd unmap.

Tarotis replied:
Are you sure? As I understand it, 'through' ("Trog") is meant, not ' through' ("durch"). Then this entry corresponds to only one concept.

AnnikaTjuka replied:
We have 205 TROUGH (Trog) and 2629 TRAY as concept sets in Concepticon. See:
https://concepticon.clld.org/parameters/205 and https://concepticon.clld.org/parameters/2629

Tarotis replied:
aaay now I understand, thank you! (:

martino-vic replied:
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534

Tarotis reviewed (changes requested):
I'll happily take over one of the review. Thank you @martino-vic for adding this concept list. Many times I find the mapping difficult because of the blurry meanings involved with the historical data.This is why I would thus propose to unmap in some cases, as you can see in my comments. I also had doubts on two cases, where I tagged @AnnikaTjuka to provide guidance.

AnnikaTjuka commented on concepticondata/concepticon.tsv:
```diff
@@ -3958,4 +3958,4 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3974	UNHAPPY	Emotions and values	Feeling not cheerful or glad, sometimes due to a misfortune.	Property	
-3975	NERVOUS	Emotions and values	A mental state of nervousness or agitation. Also, being easily excited or irritated.	Property	
\ No newline at end of file
+3975	NERVOUS	Emotions and values	A mental state of nervousness or agitation. Also, being easily excited or irritated.	Property	
```
It could be that the file is marked as changed because an empty line was added at the end. @martino-vic Could you check the difference between the `concepticon.tsv` on the master branch and on your fork? Maybe with `git diff`.

martino-vic replied:
I have no idea what leads to this strange behaviour, I mentioned it in https://github.com/concepticon/concepticon-data/pull/1236#issuecomment-1268597875 already. Happens now constantly somehow.

AnnikaTjuka commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-170	170	to catch fire, be kindled, to be ignited		
+RonaTas-2011-431-171	171	to knead, pug	274	KNEAD
+RonaTas-2011-431-172	172	to assemble, gather	7	GATHER
+RonaTas-2011-431-173	173	to collect,gather something		
```
See above.

martino-vic replied:
Hm. I'm not sure. The previous word is a fossilised reflexive form and means "to gather" but in a reflexive sense, like "a crowd gathered". Where as this one is a fossilised active form, incidentally with the exact same polysemy as English <collect>. Not sure how to map this.

AnnikaTjuka replied:
If the previous gloss _to assemble, gather_ RonaTas-2011-431-172 means "a crowd gathered" it needs to be mapped to 3638 ASSEMBLE (GET TOGETHER). Then the gloss _to collect,gather something_ can be mapped to 7 GATHER.

martino-vic replied:
Resolved in 2c64368e0fcb7cc87964075b2696cbf49d3f68ba

AnnikaTjuka commented:
Many thanks for the review, @Tarotis! I left some comments. @martino-vic In some cases, it would be helpful if you could give us a bit more detail about the list to disambiguate the mappings.

martino-vic commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-210	210	damage, loss	2947	DAMAGE (INJURY)
+RonaTas-2011-431-211	211	man of grit, stout fellow		
+RonaTas-2011-431-212	212	sheepfold, cattlegrid, stockyard, a yard encircled by a fence, shed for animals		
+RonaTas-2011-431-213	213	pink, several kinds of pink, Danthus, D. caryophyllus, several members of the Caryophillaceæ	3446	PINK
```
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534

martino-vic commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-236	236	wild/meadow saffron, Colchicum autumnale; lions tooth, Taraxacum	2230	UNTAMED OR WILD
+RonaTas-2011-431-237	237	a bird similar to a falcon, a kind of bird of prey		
+RonaTas-2011-431-238	238	pain	1783	PAIN
+RonaTas-2011-431-239	239	gourd, calabash, Lagenaria vulgaris; head, billy-cock-hat	1256	HEAD
```
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534

martino-vic commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-271	271	a kind of dog, a Hungarian breed of sheepdog		
+RonaTas-2011-431-272	272	a fish living at the bottom of the river, gudgeon, Gobio fluviatilis		
+RonaTas-2011-431-273	273	laughing bird, Picus viridis		
+RonaTas-2011-431-274	274	dry summer fog, rainbow	1733	RAINBOW
```
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534

martino-vic commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-348	348	freckle, sun spot, stain	530	FRECKLE
+RonaTas-2011-431-349	349	part of a village, street, group of people; row, line, order	1362	STREET
+RonaTas-2011-431-350	350	threshing floor (round)		
+RonaTas-2011-431-351	351	fumes, vapour, spirits, alcohol, humours	404	ALCOHOL (FERMENTED DRINK)
```
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534

martino-vic commented:
Hearty thanks @Tarotis and @AnnikaTjuka for taking the effort to look through these quite challenging cases! I've tried to work myself through them as much as I could and give extra explanations where appropriate. I think most cases are practically closed, but some remain open to debate. One thing I still have to do is to [re-numerate the column NUMBER](https://github.com/concepticon/concepticon-data/pull/1240#discussion_r991600969) and rename the file to `430`

LinguList commented on concepticondata/conceptlists.tsv:
```diff
@@ -408,4 +408,4 @@ KochGrunberg-1914-821	Koch-Grünberg, Theodor	 1914		821	annotated, areal, basic
 Epps-2021-843	Van Epps, Briana, and Carling, Gerd and Sapir, Yair	2021		843	documentation, historical, norms	English	Old Norse, Norwegian, Old Swedish, Swedish, Jamtlandic, Elfdalian	https://github.com/gerdcarling/gendernorthscandinavian	Epps2021		This list is derived from of a study focusing on lexical gender assignment in six North Scandinavian varieties. It provides statistical information abo ...
 Nordenskioeld-1905-50	Nordenskiöld, Erland	1905		50	basic, areal	German	South American languages	https://www.pueblos-originarios.ucb.edu.bo/Record/106001946?lng=da	Nordenskioeld1905	Nordenskioeld1905	This list compares five languages of the Madre de Dios region in the Peruvian selva across 50 basic concepts. While four languages are part of the hypothesized Pano-Takana language family, one langua ...
-
+RonaTas-2011-431	Róna-Tas, András, and Berta, Árpád	2011		431	historical	English	Hungarian, Old Hungarian, Early Ancient Hungarian, West Old Turkic, Cumanian		RonaTas2011		This list features 431 Hungarian words that were loaned from West Old Turkic or Cumanian
```
Every NOTE in the conceptlists.tsv file needs to end in a dot.

martino-vic replied:
Resolved in 6e464ab0091928c4ac74a148867e3a8cdb17b762

martino-vic replied:
Also changed "Cumanian" to "Cuman". The dictionary uses both, but Wikipedia uses the latter.

LinguList commented on concepticondata/references/references.bib:
```diff
@@ -4266,5 +4266,14 @@ @Article{Nordenskioeld1905
+@Book{RonaTas2011,
+  Author                              = {Róna-Tas, András, and Berta, Árpád},
+  Title                               = {West Old Turkic},
+  Subtitle                            = {Turkic loanwords in Hungarian}
```
We don't use the submtitle category in Concepticon. Just add the subtitle with a dot to the title.

martino-vic replied:
Resolved in c6dbb41d1118eab6a1571917fa1d6db0b9a01c7c
Also fixed a missing comma.

LinguList reviewed (commented):
See my comment,s I hope they are helpful.

mathildavz commented:
I'll happily review this PR. I have some other tasks to get to first, but I'll do my best to provide a review by the end of the week. Is that soon enough or should I prioritize this, @AnnikaTjuka?

AnnikaTjuka commented:
> Is that soon enough or should I prioritize this, @AnnikaTjuka?

That's soon enough, @mathildavz.

martino-vic commented:
Thanks @LinguList for the very useful comments. I tried to implement the suggestions and marked the conversations as resolved.

If all changes prove to be fine, the only question that is left open now is whether a column with Hungarian words should be added or not (https://github.com/concepticon/concepticon-data/pull/1240#discussion_r992205996). Plus comments by @mathildavz, which will come on Friday.

mathildavz commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,431 @@
+RonaTas-2011-431-141	141	pelican, Pelicanus	472	PELICAN
+RonaTas-2011-431-142	142	polecat, Putorius foetidus		
+RonaTas-2011-431-143	143	matchmaker		
+RonaTas-2011-431-144	144	gleaner mouse, Mus spicilegus		
```
Maybe this can be mapped to 1491 MOUSE?

martino-vic replied:
Same issue as above about hypernyms

mathildavz commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,431 @@
+RonaTas-2011-431-213	213	pink, several kinds of pink, Danthus, D. caryophyllus, several members of the Caryophillaceæ
+RonaTas-2011-431-214	214	stake, pale, stick	1295	STICK
+RonaTas-2011-431-215	215	a fishing water bird with black feathers, cormorant,Phalacrocorax carbo	840	CORMORANT
+RonaTas-2011-431-216	216	sparrow hawk, Accipiter n. nisus		
```
Maybe this could be mapped to 307 HAWK. This is the German "Sperber" so it's not exactly a hawk, but they belong to the same family and even the same genus (Accipiter).

martino-vic replied:
see issue about again

mathildavz commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,431 @@
+RonaTas-2011-431-353	353	thin hide rope, strap (used for embroidery or as a whip), refrozen thin layer after the melting of snow	2973	STRAP
+RonaTas-2011-431-354	354	rock, cliff, occiput, crest (of mountain)
+RonaTas-2011-431-355	355	word	1599	WORD
+RonaTas-2011-431-356	356	a kind of falcon, Pielfalcus, Gyrfalco		
```
Maybe this could be mapped to 3258 FALCON?

martino-vic replied:
again the issue above

mathildavz reviewed (commented):
Here is my review. I hope my comments are helpful.

martino-vic commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,431 @@
+RonaTas-2011-431-5	5	false, imitation	3069	FALSE
+RonaTas-2011-431-6	6	pied (of an animals coat)		
+RonaTas-2011-431-7	7	apple	1320	APPLE
+RonaTas-2011-431-8	8	barrel (wooden container andstandard quantity)	2548	BARREL
```
Ah, well spotted. Apparently I haven't accounted for line breakes in my pdf-parser because most of the meanings were short enough to be on the same line, so I didn't detect this shortcoming.

martino-vic commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,431 @@
+RonaTas-2011-431-32	32	to estimate, to esteem, to appreciate		
+RonaTas-2011-431-33	33	frog	503	FROG
+RonaTas-2011-431-34	34	hobble, shackle, leg-iron		
+RonaTas-2011-431-35	35	stamp, bond, mark	1720	SIGN
```
Ah wait, the original Hungarian  word <bélyeg> actually corresponds to `596	POSTAGE STAMP`. The other two meanings are archaic or dialectal or sth like that.

martino-vic commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,431 @@
+RonaTas-2011-431-35	35	stamp, bond, mark	1720	SIGN
+RonaTas-2011-431-36	36	wage, rent		
+RonaTas-2011-431-37	37	name of a people		
+RonaTas-2011-431-38	38	grain or small globule of preciousmetal	605	GRAIN
```
Ah same issue as in https://github.com/concepticon/concepticon-data/pull/1240#discussion_r993464108. Will unmap then.

martino-vic commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,431 @@
+RonaTas-2011-431-47	47	to forgive	1528	FORGIVE
+RonaTas-2011-431-48	48	buckler with handle, handle of different arms		
+RonaTas-2011-431-49	49	knot, bend	760	KNOT
+RonaTas-2011-431-50	50	stack of hay		
```
Same issue as in https://github.com/concepticon/concepticon-data/pull/1240#discussion_r993469599. Except that I'm not sure if "stack of hay" really is a hypernym of "hay". I'd maybe even rather map it to `2984	HEAP` if anything

martino-vic commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,431 @@
+RonaTas-2011-431-58	58	cover	761	COVER
+RonaTas-2011-431-59	59	calf	1989	CALF
+RonaTas-2011-431-60	60	pepper	1385	PEPPER
+RonaTas-2011-431-61	61	pea, Pisum sativum; bean	832	BEAN
```
Ah yes, the original word is <borsó> which means "pea". "bean" is just a secondary meaning, again archaic or dialectal or something like that.

martino-vic commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,431 @@
+RonaTas-2011-431-75	75	Ismaelite, name of an ethnic group in Medieval Hungary		
+RonaTas-2011-431-76	76	sad, bad-tempered	699	SAD
+RonaTas-2011-431-77	77	farewell, indulgence, dedication festival, annual parish-feast	3554	GOODBYE
+RonaTas-2011-431-78	78	ox; also ox, goat, ram without or with small or unusual horns;stupid
```
See discussion at https://github.com/concepticon/concepticon-data/pull/1240#discussion_r991335684

martino-vic commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,431 @@
+RonaTas-2011-431-84	84	sin	2971	SIN
+RonaTas-2011-431-85	85	(blunt) end of something, a rod with an end cut, obtuse end of a field-strip, the shorter side or end of a piece of furniture, foot of the hill		
+RonaTas-2011-431-86	86	magical		
+RonaTas-2011-431-87	87	a kind of willow, Salix elægnos		
```
Again, no idea how hypernyms should be treated. According to https://github.com/concepticon/concepticon-data/pull/1240#discussion_r991354313 for example I should leave them unmapped

martino-vic commented:
One big question seems to have crystallised out of all these smaller issues, namely how to treat hypernyms. Say I have "a type of X", do I map it to concept "X" ?

LinguList commented:
@martino-vic, the core of concept mapping is that you NEVER map `a type of x` to `x`.

LinguList commented:
I just replied there.

LinguList commented:
My suggestion is: leave Hungarian out for now.

martino-vic commented:
So just to double-check: Is there anything left for me to do now?

AnnikaTjuka commented:
@martino-vic Could you please run the `concepticon test` again? The checks seem to fail again.

martino-vic commented:
Had `ERROR:: conceptlist missing in conceptlists.tsv: ignore.tsv`.  I had placed the ignore.tsv (https://github.com/concepticon/concepticon-data/pull/1240#discussion_r991811377) into the wrong folder. Fixed this in dc9e1532a9642e20e3d51b3114286d8c69e925d7. Now the tests should pass, since I get:

testing 410 concept lists
INFO    all integrity tests passed: OK

LinguList commented:
Nice, then merge, once you got approved and green light from the others, 
many thanks!

AnnikaTjuka commented:
@Tarotis Could you check that all the changes you requested have been implemented and approve the PR if so? I'll do a final check afterward.

Tarotis commented on concepticondata/conceptlists.tsv:
```diff
@@ -408,4 +408,4 @@ KochGrunberg-1914-821	Koch-Grünberg, Theodor	 1914		821	annotated, areal, basic
 Epps-2021-843	Van Epps, Briana, and Carling, Gerd and Sapir, Yair	2021		843	documentation, historical, norms	English	Old Norse, Norwegian, Old Swedish, Swedish, Jamtlandic, Elfdalian	https://github.com/gerdcarling/gendernorthscandinavian	Epps2021		This list is derived from of a study focusing on lexical gender assignment in six North Scandinavian varieties. It provides statistical information abo ...
 Nordenskioeld-1905-50	Nordenskiöld, Erland	1905		50	basic, areal	German	South American languages	https://www.pueblos-originarios.ucb.edu.bo/Record/106001946?lng=da	Nordenskioeld1905	Nordenskioeld1905	This list compares five languages of the Madre de Dios region in the Peruvian selva across 50 basic concepts. While four languages are part of the hypothesized Pano-Takana language family, one langua ...
-
+RonaTas-2011-431	Róna-Tas, András, and Berta, Árpád	2011		431	historical	English	Hungarian, Old Hungarian, Early Ancient Hungarian, West Old Turkic		RonaTas2011		This list features 431 Hungarian words that were loaned from West Old Turkic, compiled by András Róna-Tas.
```
I think the comma after "András" has to go in order to be correctly in the bibtex format, right?

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,431 @@
+RonaTas-2011-431-27	27	label, tag		
+RonaTas-2011-431-28	28	cattle, livestock	1153	CATTLE
+RonaTas-2011-431-29	29	velvet		
+RonaTas-2011-431-30	30	to fuck		
```
Before we decide to not map this to `COPULATE`, what does the actual Hungarian original mean? Similar to `barrow`, it seems like we should not judge from the English translation, but rather, as you say, from the "modern Hungarian meaning"

martino-vic replied:
Well okay, I am still in the process of understanding how classifying these words works. Judging from this https://github.com/concepticon/concepticon-data/pull/1242#discussion_r994720971 and looking at preexisting mappings in concepticon, there are actually 4 cases, all in [Scheible 2014 1755](https://concepticon.clld.org/contributions/Scheible-2014-1755), where German versions of "to fuck" are mapped to `COPULATE`. To me, it still somehow feels wrong because I feel like one is a subcategory of the other, and according to this https://github.com/concepticon/concepticon-data/pull/1240#issuecomment-1276575304, that should never be the case. According to this https://github.com/concepticon/concepticon-data/pull/1242#discussion_r994733533 however, precedence cases within Concepticon trump that principle. Which in this case would then mean that I'd actually map it to `COPULATE`.

Tarotis commented on concepticondata/conceptlists/RonaTas-2011-431.tsv:
```diff
@@ -0,0 +1,431 @@
+RonaTas-2011-431-93	93	only, just	1580	ONLY
+RonaTas-2011-431-94	94	pick-axe; mace; a small axe; war hammer		
+RonaTas-2011-431-95	95	nettle, Urtica	33	NETTLE
+RonaTas-2011-431-96	96	thicket, brushwood, dry foliage, grove, meadow	2819	GRASSLAND
```
Didn't we decide to leave the duplicate line in there? Also, I wouldn't map to Grassland but rather unmap.

martino-vic replied:
Ah, well spotted, yes.

Tarotis commented on concepticondata/references/references.bib:
```diff
@@ -4266,5 +4266,13 @@ @Article{Nordenskioeld1905
+}
+
+@Book{RonaTas2011,
+  Author                              = {Róna-Tas, András, and Berta, Árpád},
```
Same as in the `conceptlist.tsv`: The comma after "András" should be removed.

Tarotis commented on concepticondata/references/references.bib:
```diff
@@ -4266,5 +4266,13 @@ @Article{Nordenskioeld1905
+
+@Book{RonaTas2011,
+  Author                              = {Róna-Tas, András, and Berta, Árpád},
+  Title                               = {West Old Turkic. Turkic loanwords in Hungarian},
```
Most citation styles separate title and subtitle by colon, not by a dot

martino-vic replied:
Ah, according to this we use dots https://github.com/concepticon/concepticon-data/pull/1240#discussion_r991811756 but if you say it's more common with a comma, I'll go for a comma then

Tarotis replied:
I didn't see that comment, but then just keep the dot :)

Tarotis reviewed (changes requested):
I added a small amount of comments, primarily with questions and some comments on the bib-file :)

martino-vic commented:
Thank you for the helpful comments @Tarotis. I implemented the recommendations during the last PR. The tests seem to be passing too.

Tarotis reviewed (approved):
Thanks for all the work put into this, green light from my side now :)

AnnikaTjuka commented on concepticondata/ignore.tsv:
```diff
@@ -0,0 +1 @@
+ID	GLOSS	COMMENT
```
Just did my final check and noticed that the entry in the `ignore.tsv` file is no longer there. @martino-vic Could you add it back?

martino-vic replied:
I removed its content as a result of this https://github.com/concepticon/concepticon-data/pull/1240#discussion_r995615044 discussion. I left the file there though, in case somebody in the future wants to fill it up, so it's already there.

AnnikaTjuka replied:
I see!

AnnikaTjuka reviewed (approved):
Thanks for your patience and for resolving all comments, @martino-vic! Feel free to merge now. And congratulations on submitting your first list!

LinguList commented:
I just merged, so @martino-vic can concentrate on the blogpost, which is important for the dissertation ;-) Thanks to all of you for the patience, great help, and enthusiasm to discuss so many details!

martino-vic commented:
Thanks everybody for the great cooperation!
